<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1247</width>
    <height>691</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Newspaper P2P sharing</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="1,0,2">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="pushButton_new_peer">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>New peer</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_add_news">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>Add news</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTreeWidget" name="treeWidget_newspaper">
              <column>
               <property name="text">
                <string>Name</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Type</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>ID</string>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QPushButton" name="pushButton_article_list">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Get Article List</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_external_article">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Preview article</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textEdit_article">
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>160</y>
          <width>264</width>
          <height>97</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPushButton" name="pushButton_load">
             <property name="text">
              <string>Load</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_testPeer2">
             <property name="text">
              <string>Test Peer 2 create</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="pushButton_testPeer1">
             <property name="text">
              <string>Test Peer 1 create</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_save">
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_loadFromFile">
             <property name="text">
              <string>Load from file</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QLineEdit" name="lineEditFriends">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>370</y>
          <width>311</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonFriends">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>370</y>
          <width>121</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Add friend</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEditFriendID">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>340</y>
          <width>441</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>20</y>
          <width>1121</width>
          <height>551</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Articles and newspapers</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6"/>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Journalists</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLineEdit" name="lineEdit_addJournalist"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_addJournalist">
               <property name="text">
                <string>Add journalist</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Margins</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QPushButton" name="pushButton_add_margin">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Add margin</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_view_margin">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>View Margin</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLineEdit" name="lineEdit_margin"/>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="text">
                  <string>Request margin</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Set IP</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout" stretch="7,1">
             <item>
              <widget class="QComboBox" name="comboBox_interfacs"/>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_set_ip">
               <property name="text">
                <string>Set IP</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label_article">
             <property name="text">
              <string>Article</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QPushButton" name="pushButton_add_article">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Add article</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_delete_article">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Delete article</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_print_peer">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>print Peer</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_margin">
             <property name="text">
              <string>Margin contents</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPlainTextEdit" name="plainTextEdit_margins"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Page</string>
       </attribute>
       <widget class="QDateTimeEdit" name="dateTimeEdit">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>550</y>
          <width>141</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QListWidget" name="listWidget_articles">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>70</y>
          <width>361</width>
          <height>421</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_news_select">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>30</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
       </widget>
       <widget class="QToolButton" name="toolButton_articleList">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>30</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>⟳</string>
        </property>
       </widget>
       <widget class="QTextBrowser" name="textBrowser">
        <property name="geometry">
         <rect>
          <x>430</x>
          <y>30</y>
          <width>771</width>
          <height>541</height>
         </rect>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox_categories">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>500</y>
          <width>361</width>
          <height>27</height>
         </rect>
        </property>
       </widget>
       <widget class="QToolButton" name="toolButton_addNewspaper">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>30</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>+</string>
        </property>
       </widget>
       <widget class="QToolButton" name="toolButton_removeNewspaper">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>30</y>
          <width>31</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>−</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_informCoworkers">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>550</y>
          <width>87</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>IC</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_gossip">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>550</y>
          <width>87</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Gossip</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1247</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
